plugins/boundary/mains/aws/go.mod

```diff
@@ -1,33 +1,33 @@
 module github.com/hashicorp/boundary/plugins/boundary/mains/aws
 
-go 1.23.1
+go 1.23.3
```

Comment:
Did you mean to update this? If so, we should just update our Go version everywhere (e.g. `.go-version`)

Reply:
That must have been go mod tidy, ill update the rest for main and use go toolchain for the release/0.18.x backport